http/file_server.ts

```diff
@@ -42,7 +42,8 @@ interface FileServerArgs {
   quiet: boolean;
 }
 
-const encoder = new TextEncoder();
+// @ts-ignore `Deno.core` is not a public API
+const encode = Deno.core.encode || new TextEncoder().encode;
```

Comment:
Not in favor of this one.

Reply:
Agreed, we should just use `TextEncoder`

Reply:
fine, comes with a performance penalty but whatever. Updated the benchmark results.